### PR TITLE
link typos

### PR DESCRIPTION
**Pull request by nitedog**
fixing small link typos (linkos)

### CODE REVIEW

File: _about/participating.md [hunk 1]
```diff
@@ -93,11 +93,11 @@ and to get information on participation and links to the mailing list
 archives.
 
 -   [ARIA Working
-    Group](https://www.w3.org/WAI/ARIA/ "link to ARIA WG home page")
```
Comment (nitedog) [inline _about/participating.md]: Not sure why we have this here and not for other links. Seems superfluous.

File: _about/participating.md [hunk 2]
```diff
@@ -93,11 +93,11 @@ and to get information on participation and links to the mailing list
...
+    Group](https://www.w3.org/WAI/ARIA/)
     develops a suite of Accessible Rich Internet Applications resources,
     and accessible APIs and mappings.
--   [[Accessible Platform Architectures (APA) Working
-    Group]{style="text-decoration: underline;"}](https://www.w3.org/WAI/APA/)
```
Comment (nitedog) [inline _about/participating.md]: Not sure what this does but the styling directives are appearing on the resulting page.

File: _about/participating.md (hunk 1 above)
Comment (yatil) [reply _about/participating.md]: Agree!

File: _about/participating.md (hunk 2 above)
Comment (yatil) [reply _about/participating.md]: Obviously this does nothing :-D I guess there were some inline-styles on the old website that was added and tried to survive the transition. Thanks for catching!

**Pull request merged**